### DISCUSSION TITLE
Vue 3 Support

### DISCUSSION THREAD

**Discussion by jewelsonn** [wevm/wagmi#2273] [Ideas]
Hey, Can you guys add support for Vue Js 3? I know there is a repo called as "vagmi" but it's pretty much outdated as it's using @wagmi/core ^0.4 and maintained by one person. He's done a great job, a merge would be really great.

**Comment by unicape** [comment]
bro look here [use-wagmi](https://github.com/unicape/use-wagmi)
Already supports most APIs and is developing for wagmi@1.0
